osm-to-route-snapper/Cargo.toml

```diff
@@ -3,11 +3,26 @@ name = "osm-to-route-snapper"
 version = "0.1.0"
 edition = "2021"
 
+[lib]
```

Comment:
For simplicity, this one crate acts in 3 ways: as a regular Rust library, as a runnable binary with the CLI, and as a WASM interface